back/src/Guide/Api/Types.hs

```diff
@@ -245,7 +246,7 @@ data TraitSite route = TraitSite
   ,  _createTrait :: route :-
       Summary "Create a new trait in the given item"
       :> Description "Returns the ID of the created trait."
-      :> ErrorResponse 400 "'content' not provided"
+      :> ErrorResponse 400 "'content' can not be empty"
```

Comment:
Would be nice to have some kind of `NotNullText` so that those would be handled automatically.

Reply:
let's open issue.

Reply:
I created an entry in https://www.notion.so/aelve/Backend-technical-debt-45a41ab060b7400bb7d0e6547fa34785